src/pages/bioclimatic/component.jsx

```diff
@@ -53,23 +96,41 @@ function BioClimaticPage(props) {
           >
             {map => (
               <LayerManager map={map} plugin={PluginMapboxGl}>
-                {activeLayers
+                {bioclimaticLayers
                   .filter(l => l.active)
                   .map(layer => (
+                    // TODO: fix all eslint-disables
                     // eslint-disable-next-line react/jsx-props-no-spreading
                     <Layer key={layer.id} {...layer} />
                   ))}
               </LayerManager>
             )}
           </Map>
-          <Timeline className="timeline" activeTab={scenario} data={timelineData} hideHeader />
-          <LayerToggle layers={activeLayers} setLayers={setActiveLayers} />
+          <Timeline
+            className={styles.timeline}
+            activeTab={scenario}
+            data={timelineData}
+            yearIndex={yearIndex}
+            handleOnChange={setYearIndex}
+            hideHeader
+          />
+          <div className={styles.navigationBar}>
+            <Zoom viewport={viewport} setViewport={setViewport} />
+            <LayerToggle theme={styles.layerToggle} />
+          </div>
           <Modal
             title="Bioclimatic variables data"
             text={`Bioclimatic variables derived from Copernicus describing temperature and precipitation annual tendencies, seasonality
                   and extreme climatic conditions, including a combination of both environmental factors for current and future scenarios.`}
             storageKey="bioclimatic"
           />
+          <RampLegend
+            title={biovarName}
+            colorRamp={['#FEF6B5', '#FFA679', '#E15383']} // purple
```

Comment:
maybe we should use vars for things like these 🤔

Reply:
Great idea! I moved them to `constants.js` file, together with Species distribution ramp :+1: